.envServer

```diff
@@ -0,0 +1,3 @@
+PORT=3000
+MONGO_PASSWORD=Gyy628\\nAWS
+MONGOO_URI=mongodb://localhost:27017
```

Comment:
I'm not sure we should commit the `.env`. Besides, even if these are just for development, let's not commit and push database credentials. WDYT?